workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFClaimList/hooks/useFilters.tsx

```diff
@@ -72,6 +117,14 @@ export default function useFilters(
 			initialFilter = initialFilter
 				? initialFilter.concat(` and (${statusFilter})`)
 				: initialFilter.concat(`(${statusFilter})`);
+
+			urlParams.delete('status');
+			filters.status.value.forEach((value) =>
```

Comment:
new line between `urlParams.delete` and `filters.status.value.forEach` as well

Reply:
Done!